lib/mayaUsd/fileio/utils/jointWriteUtils.cpp

```diff
@@ -237,6 +237,16 @@ int UsdMayaJointUtil::getCompressedSkinWeights(
     unsigned int numInfluences;
     skinCluster.getWeights(outputDagPath, components.object(), weights, numInfluences);
 
+    if (weights.length() < numVertices * numInfluences) {
+        MString msg("Number of weights (");
+        msg += weights.length();
+        msg += ") exceeds number of influences * number of vertices (";
+        msg += numInfluences * numVertices;
+        msg += ") - skin binding will not be written";
+        MGlobal::displayError(msg);
```

Comment:
Would it be possible to add to this message a way to fix the problem or a workaround for problem?